posts/7-common-errors-to-check-when-debugging-airflow-dag.md

```diff
@@ -1,223 +1,203 @@
-
-**5. You can exec into one of your Celery workers to look for the log files there**
-
-This functionality is for Enterprise-only customers or folks using Kubernetes.
-
-Once you're [set up with Kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/), you can run: `kubectl exec -it {worker_name} bash`
-
-Log files should be in `~/logs`. From there, they'll be split up by DAG/TASK/RUN.
-
-### 7. Tasks are slow to schedule and/or have stopped being scheduled altogether
-
-   If your tasks are slower than usual to get scheduled, you'll want to check how often you've set your scheduler to restart. Airflow has an unfortunately well-known problem by which the scheduler's performance degrades over time and requires a quick restart to ensure optimal performance.
-
-   The frequency of restarts is defined in your airflow.cfg as `"run_duration"`. A `run_duration` of -1 indicates that you never want your Scheduler to restart, whereas a `run_duration` of 3600 will restart your scheduler every hour. Check out [this forum post](https://forum.astronomer.io/t/how-can-i-schedule-automatic-scheduler-restarts/61) for more info. We generally restart our own schedulers about once a day, but the frequency at which you might is very much dependent on your particular use case. 
-
-   If you're using Astronomer, you can restart your scheduler by either:
+If you're using Astronomer, you can restart your scheduler by either:
 
 1. Inserting `AIRFLOW__SCHEDULER__RUN_DURATION={num_seconds_between_restarts} `as an Environment Variable in the Configure page of the Astronomer UI to set a recurring restart OR
 
 2. Running `astro airflow deploy` via your CLI to immediately restart everything (if you're running Celery, there is a [Worker Termination Grace Period](https://forum.astronomer.io/t/what-is-the-worker-termination-grace-period-on-astronomers-ui/141) you can leverage here to minimize existing immediate task disruption)
-**Was this helpful?**
+
+> **Note:** Scheduler performance was a critical part of the [Airflow 2.0 release](https://www.astronomer.io/blog/introducing-airflow-2-0) and has seen significant improvements since December of 2020. For more information, read [The Airflow 2.0 Scheduler](https://www.astronomer.io/blog/airflow-2-scheduler). 
+
+## Was this helpful?
 
 This list is based on our experience helping Astronomer customers with core Airflow issues, but we want to hear from you. Don't hesitate to reach out to us at humans@astronomer.io if we missed something that you think would be valuable to include.
 
-If you have follow up questions or are looking for Airflow support from our team, reach out to us [here](https://www.astronomer.io/contact/?from=/).
+If you have follow up questions or are looking for Airflow support from our team, [reach out to us](https://www.astronomer.io/contact/?from=/).
 <!-- markdownlint-disable-file -->
```

Comment:
Add new line at the end

Reply:
@sunkickr Is this just add another line between 202 and `<!-- markdownlint-disable-file -->`? Just did that - let me know if that works.

Reply:
I think its anew line after <!-- markdownlint-disable-file -->

Reply:
Ah ok... I think I got it?